### PR TITLE
fix: Corrects logic where it was possible to customize avatar

### LINKED ISSUE #6427: Able to select locked avatar skins / hairstyles / shirts without purchasing with gems.
User ID: f96fd5ef-3a77-4913-8d2f-ec8af90b77b2
Browser: Firefox
OS: Windows 7

I'm a programmer so I prod things. I noticed that one is able to "select" locked avatar styles.

I've also tested shirts and skin colors, so I assume the bug is general to avatar customization in general.

### PR DESCRIPTION
**Pull request by crookedneighbor**
with gem purchasable items without purchasing them.

Closes #6427

### CODE REVIEW

File: common/script/content/appearance.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  },
+  skin: {
+    ddc994: true,
+    f5a76e: true,
+    ea8349: true,
+    c06534: true,
+    '98461a': true,
+    915533: true,
+    c3e1dc: true,
+    '6bd049': true,
```
Comment (Alys) [inline common/script/content/appearance.js:59]: there's a mix of quoted and unquoted values here

Comment (crookedneighbor) [inline common/script/content/appearance.js:59]: As stupid as it is, object attributes that begin with a number and contain letters must be quoted. Those that begin with a letter or properties that are all numbers do not. I can quote them all and disable the eslint rule that objects to unnecessarily quoted items, if you prefer. I was debating about it.

Comment (Alys) [inline common/script/content/appearance.js:59]: Ah, sorry. The inconsistency looks weird, but there's a good reason for it. Disabling that rule is probably undesirable. I'm happy with whatever you want there.

Comment (crookedneighbor) [inline common/script/content/appearance.js:59]: I think you're right though, it looks weird. I'll set the style rule to be ignored just for this section.

**Pull request closed without merging**